index.bs

```diff
@@ -31,6 +30,42 @@ urlPrefix: https://w3c.github.io/sensors; spec: GENERIC-SENSOR
     text: construct a sensor object; url: construct-sensor-object
 </pre>
 
+
+<pre class=biblio>
```

Comment:
Could you also inline biblio.json? If I remember correctly, online generator doesn't work with external biblio files.